ooiui/templates/science/plotting.html

```diff
@@ -434,17 +428,44 @@
         }
       }
     });
+    
+    //render the time series data
+    this.listenTo(this.collections.dataseries, 'reset', function(series) {
+      this.views.timeseriesView.collection = series;
+      this.views.timeseriesView.render();
+
+      self.collections.annotations.each(function(annotation,i) {  
+        self.views.timeseriesView.views.highchartsView.chart.xAxis[0].addPlotBand({
+                from: moment.utc(annotation.get('beginDT')),
+                to: moment.utc(annotation.get('endDT')),
+                color: '#FCFFC5',
```

Comment:
@Bobfrat , @birdage lol may want to double check the removal of time data from the streams doesn't break this.